### ISSUE TITLE
WearHF doesn't work correctly with Flutter accessibility mechanism and Semantics

### ISSUE THREAD

**Issue by Skyggedans** [realwear/HTML#8]
Out of the box, WearHF allows you to activate by voice simple buttons only.
And even if you wrap your widgets with Semantics, it won't work in recent firmware versions.
If you're interested how to work around this, please check my Gradle plugin that fixes this issue during build process using byte code manipulation:
[https://plugins.gradle.org/plugin/com.rockwellits.patchflutteraccessibility](url)
With this one, you can wrap by Semantics virtually any widget, like Row, Card or InkWell, set its value to desired WearHR metadata/command, and you go.
